src/pages/zh-cn/guides/typescript.md

```diff
@@ -79,5 +98,101 @@ const { greeting = 'Hello', name } = Astro.props
 <h2>{greeting}, {name}!</h2>
 ```
 
+### 常见的 Props 类型的套路
+
+- 如果你的组件没有任何的参数或插槽，你可以使用 `type Props = Record<string, never>`。
+
+- 如果你的组件必须将一个子组件传递给默认插槽，你可以使用 `type Props = { children: any; };`。
+
+## 类型工具包
+
+<Since v="1.6.0" />
+
+Astro comes with some built-in utility types for common prop type patterns. These are available under the `astro/types` entrypoint.
+
+Astro 为常见的组件参数的类型模式准备了一些实用的类型工具集。这些可以通过在代码中引入 `astro/types` 来使用。
+
+### 内置的属性类型
+
+Astro 提供 `HTMLAttributes` 类型，以检查你的类型是否使用有效的 HTML 属性。你可以使用这些类型来帮助构建组件参数。
+
+例如，如果你正在构建一个 `<Link>` 组件，你可以通过以下语法来为组件的 Props 类型镜像 `<a>` 的默认的 HTML 属性。
+
+```astro title="src/components/Link.astro" ins="HTMLAttributes" ins="HTMLAttributes<'a'>"
+---
+type Props = astroHTML.JSX.AnchorHTMLAttributes;
+const { href, ...attrs } = Astro.props;
+---
+<a {href} {...attrs}>
+  <slot />
+</a>
+```
```

Comment:
I think this code example is still showing the old version. The new example looks more like this:

````suggestion
```astro title="src/components/Link.astro" ins="HTMLAttributes" ins="HTMLAttributes<'a'>"
---
import { HTMLAttributes } from 'astro/types'
// use a `type`
type Props = HTMLAttributes<'a'>;
// or extend with an `interface`
interface Props extends HTMLAttributes<'a'> {
  myProp?: boolean;
}
const { href, ...attrs } = Astro.props;
---
<a {href} {...attrs}>
  <slot />
</a>
```
````

Would be good to translate the “use a `type`” and “or extend with and `interface`” comments.